app/src/main/java/com/example/trainingintive/presentation/SplashViewModel.kt

```diff
@@ -30,11 +30,16 @@ class SplashViewModel @Inject constructor(
             )
     }
 
-    override fun onCleared() {
-        super.onCleared()
-        disposables.dispose()
+    private fun checkIfUserIsLoggedAndSendEventToNavigator() {
+        if (isUserLogged()) {
+            navigator.sendEvent(SplashScreenEvent.DisplayUserScreen)
+        } else {
+            navigator.sendEvent(SplashScreenEvent.DisplayLogInForm)
+        }
     }
 
+    private fun isUserLogged() = FirebaseAuth.getInstance().currentUser != null
```

Comment:
Should be adjusted to Clean Architecture (like in previous PR, you can leave a TODO for this, for now)